### ISSUE TITLE
drag only half way

### ISSUE THREAD

**Issue by kisubs** [octopitus/rn-sliding-up-panel#8]
Hello, I've been trying to use this library, but the panel seems to come all the way to the top.
Is there a way to drag it only half way from the bottom?

Thanks!

**Comment by octopitus** [comment]
```js
const {height} = Dimensions.get('screen')

**Comment by kisubs** [comment]
This is what I have:

```
const window = Dimensions.get('window');
<SlidingUpPanel
    ref={c => this._panel = c}
    visible={true}
    draggableRange={{top: window.height * 0.5, bottom: 0}}
    allowDragging={true}
    >
    <View style={{flex: 1, backgroundColor: "yellow"}}>
        <Text>Here is the content inside panel</Text>
    </View>
</SlidingUpPanel>
```

 and its displaying:
![screenshot_20170623-091324](https://user-images.githubusercontent.com/24979247/27490763-6acb641c-57f4-11e7-9ff8-842952609586.png)

dragged up:
![screenshot_20170623-091330](https://user-images.githubusercontent.com/24979247/27490764-6ae2f226-57f4-11e7-9308-98c53a2d5af8.png)

**Comment by octopitus** [comment]
Hi @kisubs, your issue was fixed in #9. Here is an example:

```js
const {height} = Dimension.get('window')
<SlidingUpPanel
    ref={c => this._panel = c}
    draggableRange={{top: window.height / 2, bottom: 0}}
    height={height / 2} // You may need to change the height of panel to half of the window
    visible>
// ...
</SlidingUpPanel>
